libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -180,6 +180,7 @@ class AC_AttitudeControl {
     // **NOTE** Using vector3f*deg(100) is more efficient than deg(vector3f)*100 or deg(vector3d*100) because it gives the
     // same result with the fewest multiplications. Even though it may look like a bug, it is intentional. See issue 4895.
     Vector3f get_att_target_euler_cd() const { return _attitude_target_euler_angle * degrees(100.0f); }
+    const Vector3f & get_att_target_euler_rad() const { return _attitude_target_euler_angle; }
```

Comment:
There's a little whitespace issue with the "&" but we can get it at some future date.